Habitica/src/com/habitrpg/android/habitica/ui/adapter/BindingAdapters.java

```diff
@@ -0,0 +1,19 @@
+package com.habitrpg.android.habitica.ui.adapter;
+
+import android.databinding.BindingAdapter;
+
+import com.habitrpg.android.habitica.helpers.MarkdownParser;
+import com.rockerhieu.emojicon.EmojiconTextView;
+
+/**
+ * Created by void on 12/25/15.
+ */
+public class BindingAdapters {
```

Comment:
we have already binding helper class https://github.com/HabitRPG/habitrpg-android/blob/develop/Habitica/src/com/habitrpg/android/habitica/ui/helpers/DataBindingUtils.java